packages/core/src/ci-environment/services/bitrise.ts

```diff
@@ -0,0 +1,35 @@
+import type { Service, Context } from "../types";
+
+const getBranch = ({ env }: Context) => {
+  const pr =
+    env.BITRISE_PULL_REQUEST === "false" ? undefined : env.BITRISE_PULL_REQUEST;
+  const isPr = Boolean(pr);
+  return isPr ? env.BITRISEIO_GIT_BRANCH_DEST : env.BITRISE_GIT_BRANCH;
+};
```

Comment:
We don't want that, we just want the BITRISE_GIT_BRANCH, no matter what.

Reply:
Can you confirm that we don't care about the pull request target branch?